torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -479,3 +479,33 @@ def test_invalid_devices(self):
                     modes=[ModuleCreationMode.MODULE_CTOR],
                 )
             )
+
+    @skip_if_lt_x_gpu(1)
+    @dist_utils.dist_init
+    def test_input_moved_to_cuda_device(self):
+        if self.rank != 0:
+            return
+        dst_worker_name = dist_utils.worker_name((self.rank + 1) % self.world_size)
+
+        # These two CPU tensors (in args and kwargs) should be implicitly moved to an appropriate cuda device.
+        t1 = torch.ones(1)
+        args = (t1, 2)
+        t2 = t1 * 2
+        kwargs = dict(word=t2)
+
+        # Only test Python nn.Module, because script module methods don't support taking kwargs.
```

Comment:
can we test the script mode without specifying kwargs?

Reply:
Added `test_input_moved_to_cuda_device_script`